include/inexor/vulkan-renderer/vkresult_to_text.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include <vulkan/vulkan_core.h>
+
```

Comment:
```suggestion

#include <string>

```

Reply:
This is outdated, I included `<string_view>` in the new file (see pushed changes later).